tests/unit/src/app/check/controllers/abandon.test.js

```diff
@@ -0,0 +1,64 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+const Controller = require("../../../../../../src/app/check/controllers/abandon");
+
+const {
+  API: {
+    PATHS: { ABANDON },
+  },
+} = require("../../../../../../src/lib/config");
+
+jest.mock();
```

Comment:
Not sure if I'm missing something here but what are we mocking? I've never seen `jest.mock()` called like this before

Reply:
removed this line